engine/apps/mobile_app/tasks.py

```diff
@@ -51,11 +51,13 @@ def notify_user_async(user_pk, alert_group_pk, notification_policy_pk, critical)
         logger.info(f"Error while sending a mobile push notification: user {user_pk} has no device set up")
         return
 
-    message = get_push_notification_message(alert_group)
     thread_id = f"{alert_group.channel.organization.public_primary_key}:{alert_group.public_primary_key}"
-    alert_title = f"Critical page: {message}" if critical else message
     number_of_alerts = alert_group.alerts.count()
 
+    alert_title = "New Critical Alert" if critical else "New Alert"
+    alert_subtitle = get_push_notification_message(alert_group)
+    alert_body = f"Status: Firing"  # TODO: is there a better spot to calculate this rather than hardcoding?
```

Comment:
not sure if there is a better way to generate the alert status here?

Reply:
I think it would be nice to have something we can reuse (I couldn't find something already there verbalizing the `alert_group.status`)